### DISCUSSION TITLE
v1.0.161

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4851] [Release Notes]
* SSH: re-allowed AES-GCM ciphers by default again 
* SSH: honour the default username setting in the quick connect field
* macOS: titlebar double-click now maximizes window
* macOS: prevent Cmd-R from reloading the window
* fixed transfers dropdown being clipped by the terminal

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.161'>v1.0.161</a>.</em>

**Comment by WhiteBite** [comment]
fixed transfers dropdown being clipped by the terminal - NOT FIXED if panel in top too
![image](https://user-images.githubusercontent.com/19576772/139560600-a0036a37-256f-49c2-87ec-5a70aece4617.png)
